guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+      caption: Federation expiration timestamp displayed as a countdown alter when less 30 days remaining.
+    - file: federation-guardian-expiry-join
+      modalImage: federation-guardian-expiry-join@2x
+      alt: A mobile wallet interface with a screen that shows a countdown timer and a message that the federation will shut down in 15 days.
+      caption: Federation expiration timestamp displayed as a data field when more than 30 days remain or the user hasn't yet joined the federation.
+
+images_federation-id:
+    - file: federation-id1
+      modalImage: federation-id1@2x
+      alt: Federation identity and version display in a wallet interface
+      caption: Consider the most important information to display to the user about the federation.
+    - file: federation-id2
+      modalImage: federation-id2@2x
+      alt: Federation descriptions display in a wallet interface
+      caption: Different federation operator may use the description fields differently. Consider this when designing your federation details screen.
+    - file: federation-id3
+      modalImage: federation-id3@2x
+      alt: Federation contact information display in a wallet interface
+      caption: Contact information is optional for the federation operator to include. If there is no contact information, consider not displaying this metadata.
+    - file: federation-id4
+      modalImage: federation-id4@2x
+      alt: Example of important announcements display in a wallet interface
+      caption: A message of the day can consists of anything the federation operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance.
+
+images_cashu-id:
+    - file: cashu-id1
+      modalImage: cashu-id1@2x
+      alt: Mint identity and version display in a wallet interface
+      caption: Consider the most important information to display to the user about the mint.
+    - file: cashu-id2
+      modalImage: cashu-id2@2x
+      alt: Mint descriptions display in a wallet interface
+      caption: Different mint operator may use the description fields differently. Consider this when designing your mint details screen.
+    - file: cashu-id3
+      modalImage: cashu-id3@2x
+      alt: Mint contact information display in a wallet interface
+      caption: Contact information is optional for the mint operator to include. If there is no contact information, consider not displaying this metadata.
+    - file: cashu-id4
+      modalImage: cashu-id4@2x
+      alt: Example of important announcements display in a wallet interface
+      caption: A message of the day can consists of anything the mint operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance.
+    - file: cashu-id5
+      modalImage: cashu-id5@2x
+      alt: Example of supported features display in a wallet interface
```

Comment:
Seems only useful for highly technical people. Can some of those be actions I can perform? Can I do a "Token state check" if that mint supports that feature?

Reply:
Yes, token state check is an action users can perform if the mint supports it. It’s quite useful because it allows you to quickly update your token status list without having to re-check each individual token. In eNuts this feature is called ‘check proofs.’ P2PK and multimint payments are also unique actions but no wallet supports multimint payments yet. To your point, listing all of this is probably not useful for most regular users so I’ll remove it.